truenas/webui
# NAS-112741 / None / Removed welcome dialog and added "Help" widget on dashboard

To test
- Make sure that the help widget shows up by default on the dashboard page
- Make sure that it's configurable and hides when unchecked on the Dashboard Configure form
- Check responsive on phone screens
